cypress/e2e/data-import/settings/unlink-from-job-profile.cy.js

```diff
@@ -0,0 +1,127 @@
+    };
+    const matchProfile = {
+      profileName: `C11116 autotest match profile ${getRandomPostfix()}`,
+      incomingRecordFields: {
+        field: '001',
+      },
+      existingRecordFields: {
+        field: '001',
+      },
+      matchCriterion: 'Exactly matches',
+      existingRecordType: EXISTING_RECORDS_NAMES.INSTANCE,
+      instanceOption: 'Admin data: Instance HRID',
+    };
+    const mappingProfile = {
+      name: `C11116 mapping profile ${getRandomPostfix()}`,
+      typeValue: FOLIO_RECORD_TYPE.INSTANCE,
+    };
+    const instanceStatusTerm = INSTANCE_STATUS_TERM_NAMES.BATCH_LOADED;
+    const catalogedDate = '###TODAY###';
+    const jobProfile = {
+      profileName: `C11116 autotest job profile ${getRandomPostfix()}`,
+      acceptedType: ACCEPTED_DATA_TYPE_NAMES.MARC,
+    };
+
+    before('create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        cy.login(userProperties.username, userProperties.password);
+
+        // create mapping profile
+        cy.visit(SettingsMenu.mappingProfilePath);
+        FieldMappingProfiles.openNewMappingProfileForm();
+        NewFieldMappingProfile.fillSummaryInMappingProfile(mappingProfile);
+        NewFieldMappingProfile.fillInstanceStatusTerm(instanceStatusTerm);
+        NewFieldMappingProfile.fillCatalogedDate(catalogedDate);
+        NewFieldMappingProfile.save();
+        FieldMappingProfileView.closeViewMode(mappingProfile.name);
+
+        // create 3 action profiles
+        cy.visit(SettingsMenu.actionProfilePath);
+        ActionProfiles.create(actionProfile1, mappingProfile.name);
+        ActionProfileView.close();
+        ActionProfiles.waitLoading();
+        ActionProfiles.create(actionProfile2, mappingProfile.name);
+        ActionProfileView.close();
+        ActionProfiles.waitLoading();
+        ActionProfiles.create(actionProfile3, mappingProfile.name);
+        ActionProfileView.close();
+        ActionProfiles.waitLoading();
+
+        // create match profile
+        cy.visit(SettingsMenu.matchProfilePath);
+        MatchProfiles.createMatchProfile(matchProfile);
+      });
+    });
+
+    after('delete test data', () => {
+      JobProfiles.deleteJobProfile(jobProfile.profileName);
+      ActionProfiles.deleteActionProfile(actionProfile1.name);
+      ActionProfiles.deleteActionProfile(actionProfile2.name);
+      ActionProfiles.deleteActionProfile(actionProfile3.name);
+      FieldMappingProfileView.deleteViaApi(mappingProfile.name);
+      MatchProfiles.deleteMatchProfile(matchProfile.profileName);
+      Users.deleteViaApi(user.userId);
+    });
+
+    it(
+      'C11116 Unlinking a match or action profile from a job profile (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        // create Job profile with linked match and action profiles
+        cy.visit(SettingsMenu.jobProfilePath);
```

Comment:
First step is open Action Profiles, not Job Profiles

Reply:
fixed